<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>notepanda</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNormalmode"/>
    <addaction name="actionSticky_note_mode"/>
    <addaction name="separator"/>
    <addaction name="actionPin_to_top"/>
    <addaction name="separator"/>
    <addaction name="actionPreview_panel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/file-add-line.svg</normaloff>:/remixicons/images/remixicons/file-add-line.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/file-3-line.svg</normaloff>:/remixicons/images/remixicons/file-3-line.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/information-line.svg</normaloff>:/remixicons/images/remixicons/information-line.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/save-line.svg</normaloff>:/remixicons/images/remixicons/save-line.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/save-2-line.svg</normaloff>:/remixicons/images/remixicons/save-2-line.svg</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/arrow-go-back-line.svg</normaloff>:/remixicons/images/remixicons/arrow-go-back-line.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/arrow-go-forward-line.svg</normaloff>:/remixicons/images/remixicons/arrow-go-forward-line.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/clipboard-line.svg</normaloff>:/remixicons/images/remixicons/clipboard-line.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/file-copy-line.svg</normaloff>:/remixicons/images/remixicons/file-copy-line.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/close-line.svg</normaloff>:/remixicons/images/remixicons/close-line.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/scissors-cut-line.svg</normaloff>:/remixicons/images/remixicons/scissors-cut-line.svg</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/printer-line.svg</normaloff>:/remixicons/images/remixicons/printer-line.svg</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/settings-4-line.svg</normaloff>:/remixicons/images/remixicons/settings-4-line.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSticky_note_mode">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/sticky-note-line.svg</normaloff>:/remixicons/images/remixicons/sticky-note-line.svg</iconset>
   </property>
   <property name="text">
    <string>Sticky note mode</string>
   </property>
   <property name="toolTip">
    <string>Sticky note</string>
   </property>
  </action>
  <action name="actionNormalmode">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/file-edit-line.svg</normaloff>:/remixicons/images/remixicons/file-edit-line.svg</iconset>
   </property>
   <property name="text">
    <string>Normal mode</string>
   </property>
   <property name="toolTip">
    <string>Normal mode</string>
   </property>
  </action>
  <action name="actionPin_to_top">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/pushpin-line.svg</normaloff>:/remixicons/images/remixicons/pushpin-line.svg</iconset>
   </property>
   <property name="text">
    <string>Pin to top</string>
   </property>
   <property name="toolTip">
    <string>Pin window</string>
   </property>
  </action>
  <action name="actionPreview_panel">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/remixicons/images/remixicons/eye-line.svg</normaloff>:/remixicons/images/remixicons/eye-line.svg</iconset>
   </property>
   <property name="text">
    <string>Preview panel</string>
   </property>
   <property name="toolTip">
    <string>Preview panel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
